admins/pageflow/entry.rb

```diff
@@ -157,6 +187,17 @@ def scoped_collection
         params.key?(:folder_id) ? result.where(folder_id: params[:folder_id]) : result
       end
 
+      def build_new_resource
+        super.tap do |entry|
+          entry.account ||=
+            AccountPolicy::Scope.new(current_user,
+                                               Pageflow::Account).entry_creatable.first ||
+            Pageflow::Account.first
```

Comment:
Indentation looks odd here. I am wondering if the `Account.first` case can occur? `new`/`create` should not be allowed if `entry_creatable` is empty, right?